packages/cli/lang/en.lyaml

```diff
@@ -1030,6 +1030,7 @@ en:
         componentCount: "Found {{ numComponents }} components in this project"
         successStatusText: "DONE"
         failedStatusText: "FAILED"
+        errorFetchingTaskStatus: "Error fetching {{ taskType }} status"
```

Comment:
I think this is fine for now, assuming that we're not failing the build/deploy when we error out while polling for build/deploy status. It might be useful to add a `logger.debug` with the full error message, so it's easier to debug any issues.